netbout-web/src/test/java/com/netbout/rest/TkPreviewTest.java

```diff
@@ -77,7 +77,7 @@ public void previewMessageBeforePost() throws Exception {
                 )
             ).printBody(),
             Matchers.allOf(
-                Matchers.containsString("<p>Here is a paragraph.</p>"),
+                Matchers.containsString("<p>Here is a paragraph.<br  /></p>"),
```

Comment:
@carlosmiranda I tried but unsuccessful
`XhtmlMatchers.hasXPath` throws `IllegalArgumentException` for  a valid html fragment
```
MatcherAssert.assertThat(
            "<p>Here is a paragraph.</p><p>New paragraph.</p>",
            XhtmlMatchers.hasXPath("/p[.='Here is a paragraph']")
        );
```

```
[Fatal Error] :1:29: The markup in the document following the root element must be well-formed.

java.lang.IllegalArgumentException: Invalid XML: "<p>Here is a paragraph.</p><p>New paragraph.</p>"
```